Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
I don't agree, sorry.

> Not a discussion I expect to win, but FWIW.

Please either quit with these caveats or quit with the comments that precede them, thanks. If you believe your opinion to be genuinely ignored: self-censor. If you think you'll be listened to: don't self-sabotage your own argument.